src/scene.lisp

```diff
@@ -8,7 +8,13 @@
    (%game-assets :initarg :assets
                  :type hash-table
                  :initform (make-hash-table :test #'equalp)
-                 :accessor assets)))
+                 :accessor assets)
+   (%active :initarg :active
+            :type boolean
+            :accessor active)
```

Comment:
@shelvick Can you expand on exactly what you want `%active` to do? I guess I'm not envisioning it as clearly as you are.

I'm thinking, with the design as it is in the PR, if you didn't want a scene to be loaded you just wouldn't "switch" to it. And if you didn't want it drawn, you'd just not call any draw functions with it.

I can see it being more useful if, instead of a single `*SCENE*`, we tracked a set of `*CURRENT-SCENES*` or something. I'm just curious what you're thinking.

Reply:
Remember: Half-baked. ;-) I think I was imagining that being a way to track which scene(s) were already loaded or currently being drawn. But in this implementation it's probably not needed.

Reply:
> Remember: Half-baked. ;-)

No problem :P

I'll just remove it for the time being.